### PR TITLE
Update command summary table

### PR DESCRIPTION
**Pull request by KSSWSept20**
- Rename UserGuideItem.java to CommandSummary.java.
- Add command summaries of newly implemented features to the command summary table in the help window.
- Break the fillCommandSummaryTable() method into a few methods to reduce the length of a single method.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ClearCommand.java [hunk 1]
```diff
@@ -11,7 +11,8 @@
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_CONFIRMATION = "Are you sure you want to clear ALL contacts? [y/n]";
+    public static final String MESSAGE_CONFIRMATION = "Are you sure you want to remove ALL persons and reservations "
+            + "in CulinaryContacts? [y/n]";
     public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
```
Comment (Tanzhiheng26) [inline src/main/java/seedu/address/logic/commands/ClearCommand.java:16]:
```suggestion
    public static final String MESSAGE_SUCCESS = "Clear successful!";
```

**Review by ziiqii — CHANGES_REQUESTED** [review]
image
 -> Clear all contacts and reservations in the address book.

Also could add a period to after the end of the brackets
image

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 2]
```diff
@@ -139,30 +139,69 @@ private void copyUrl() {
...
+        guideItems.add(new CommandSummary("alist", "View the archive  list.\n"));
+    }
+
+    /**
+     * Fills the command summary table with reservation-related commands.
+     * These include 'rsv', 'rsvdel', 'rsvlist', 'rsvsort'.
+     */
+    public void fillReservationRelatedCommandSummary() {
+        guideItems.add(new CommandSummary("rsv", "Make a reservation based on the name of the contact"
+                + "with the specified index.\n"
```
Comment (Tanzhiheng26) [inline src/main/java/seedu/address/ui/HelpWindow.java:198]: Add a reservation for the person at the specified index

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 3]
```diff
@@ -139,30 +139,69 @@ private void copyUrl() {
...
+     * These include 'archive', 'unarchive', 'alist'.
+     */
+    private void fillArchiveRelatedCommandSummary() {
+        guideItems.add(new CommandSummary("archive", "Archive a contact.\n"
+                + "archive INDEX\n"
+                + "e.g., archive 1"));
+        guideItems.add(new CommandSummary("unarchive", "Unarchive a contact.\n"
+                + "unarchive INDEX\n"
+                + "e.g., unarchive 1"));
+        guideItems.add(new CommandSummary("alist", "View the archive  list.\n"));
```
Comment (Tanzhiheng26) [inline src/main/java/seedu/address/ui/HelpWindow.java:189]:
```suggestion
        guideItems.add(new CommandSummary("alist", "View the archive list.\n"));
```

**Review by Tanzhiheng26 — APPROVED** [review]
LGTM, just a few nits to fix

**Review by ziiqii — APPROVED** [review]
LGTM!

**Pull request merged**